install.js

```diff
@@ -53,29 +53,26 @@ if (parentPackage &&
 }
 
 if (process.env.CLDR_URL) {
-  srcUrl = srcUrl.replace(
-    "http://www.unicode.org/Public/cldr",
-    process.env.CLDR_URL.replace(/\/$/, "")
-  );
-
+  // Load custom urls.json file from env variable.
+  srcUrl = process.env.CLDR_URL;
```

Comment:
Yes, it should have another name than in the patch. But that's already what the patch is doing, isn't it. Probably the base URL should not be hardcoded as well, but that would break compatibility with existing json files...